templates/patroni.yml.j2

```diff
@@ -47,7 +47,7 @@ bootstrap:
       {% endfor %}
  
  #      recovery_conf:
-#        restore_command: cp ../wal_archive/%f %p
+#        restore_command: wal-g wal-fetch "%f" "%p"
```

Comment:
You must not override restore_command parameter in the patroni.yml.j2 template.

Reply:
Not override?

Reply:
you need to create new variable `postgresql_recovery_conf` in the file /vars/main.yml
```
postgresql_recovery_conf: []
# - {option: "restore_command", value: "wal-g wal-fetch "%f" "%p""}
# - {option: "", value: ""}
```
If it's complicated. I can do it myself.